src/test/domains/Domain.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2022 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity 0.6.12;
+
+import {stdJson} from "forge-std/StdJson.sol";
+import {ChainlogAbstract} from "dss-interfaces/Interfaces.sol";
+import {Vm} from "forge-std/Vm.sol";
+
+contract Domain {
+
+    using stdJson for string;
+
+    string public config;
+    string public name;
+    Vm public vm;
+    uint256 public forkId;
+    uint256 public live = 0;
+
+    event Log(string, string);
+    event Log(string, uint256);
```

Comment:
I think these were left in for debug by accident.

Reply:
https://github.com/makerdao/spells-goerli/pull/132/commits/28d3dc4e0d8870cd0b47213b3a1b24014d9f2035